portal-api/api-tests/api-test-users.js

```diff
@@ -44,10 +44,10 @@ const banks = {
 
 const testUsers = [
   {
-    username: 'no-roles', email: 'no-roles', password: 'P@ssword1234', roles: []
+    username: 'no-roles', email: 'no-roles@ukexportfinance.gov.uk', password: 'P@ssword1234', roles: []
```

Comment:
can we use a different domain? i.e. `no-roles@email.com` - or something like that

Reply:
Might be prudent to do this in a separate PR since there are myriads of files with our domain?